components/bus/message_bus_test.go

```diff
@@ -0,0 +1,41 @@
+package bus_test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/ThreeDotsLabs/watermill"
+	"github.com/ThreeDotsLabs/watermill/components/bus"
+	"github.com/ThreeDotsLabs/watermill/message"
+)
+
+func TestNewEventProcessor(t *testing.T) {
```

Comment:
`TestNewMessageBus`. Also, do we have tests for the functioning of the `MessageBus`? 
`publisher_test.go` looks like it provides some, but it uses `cqrs.EventBus` and not `MessageBus`.